admin/users/views.py

```diff
@@ -132,47 +124,30 @@ def get_object(self, queryset=None):
 class ResetPasswordView(OSFAdmin, FormView):
     form_class = EmailResetForm
     template_name = 'users/reset.html'
-
-    def get(self, request, *args, **kwargs):
-        try:
-            return super(ResetPasswordView, self).get(request, *args, **kwargs)
-        except AttributeError:
-            return page_not_found(
-                request,
-                AttributeError(
-                    'User with id "{}" not found.'.format(
-                        self.kwargs.get('guid')
-                    )
-                )
-            )
-
-    def post(self, request, *args, **kwargs):
-        try:
-            return super(ResetPasswordView, self).post(request, *args, **kwargs)
-        except AttributeError:
-            return page_not_found(
-                request,
-                AttributeError(
-                    'User with id "{}" not found.'.format(
-                        self.kwargs.get('guid')
-                    )
-                )
-            )
+    context_object_name = 'user'
 
     def get_context_data(self, **kwargs):
         user = User.load(self.kwargs.get('guid'))
         try:
             self.initial.setdefault('emails', [(r, r) for r in user.emails])
         except AttributeError:
```

Comment:
https://en.wikipedia.org/wiki/List_of_HTTP_status_codes#409

> ##### 409 Conflict
> Indicates that the request could not be processed because of conflict in the request

My experience from API work is that this "conflict in the request" is frequently that some of the data passed in conflicts with assumptions about said data -- such as a passed-in email and id not being associated with the same user, though both data are assumed to map to the same user.

That being said, probably not a big deal.